src/app/SearchEditor.jsx

```diff
@@ -70,33 +75,42 @@ export default class SearchEditor extends React.Component {
         {
           this.props.searches.editableSearchLoading || !this.props.searches.editableSearch ?
           <p>Loading Saved Search...</p> :
-          <div style={styles.base}>
-            <div style={styles.filters}>
-              <UserFilters
-                editMode={true}
-                onChange={this.onFilterChange.bind(this)} />
-            </div>
-            <div style={styles.rightPanel}>
-              <Button
-                onClick={this.confirmSave.bind(this)}
-                category="primary"
-                style={styles.saveButton}>
-                Update Search <FaFloopyO style={styles.saveIcon} />
-              </Button>
-              <div style={styles.userListContainer}>
-                <UserList
-                  total={this.props.searches.userCount}
-                  onPagination={this.onPagination.bind(this)}
-                  loadingQueryset={this.props.searches.loadingQueryset}
-                  users={this.props.searches.users}
-                  userSelected={this.updateUser.bind(this)} />
-                <UserDetail
-                  breakdown={this.props.filters.breakdown}
-                  specificBreakdown={this.props.filters.specificBreakdown}
-                  commentsLoading={this.props.comments.loading}
-                  user={this.props.users.selectedUser}
-                  comments={this.props.comments.items}
-                  style={styles.userDetail} />
+          <div>
+            {/*
+              can't update name or description just yet
```

Comment:
Why is this?  When writing complex passages of code like this, providing narrative comments helps in a few ways.  Let's read through what's going on here and comment it together during the review.